test/unit-complete/main.tf

```diff
@@ -10,9 +10,17 @@ variable "aws_region" {
   default     = "us-east-1"
 }
 
+terraform {
+  required_providers {
+    # replica support was added in 2.58
+    # 3.5.0 to 3.6.0 is broken due to https://github.com/terraform-providers/terraform-provider-aws/issues/15115
+    aws = ">= 2.58, < 4.0, != 3.5.0, != 3.6.0"
```

Comment:
Oh nice! Actually, I didn't even know that this is possible!

Reply:
too sad it is needed...